src/particleContainer/adapter/RDFCellProcessor.cpp

```diff
@@ -55,6 +62,13 @@ void RDFCellProcessor::processCellPair(ParticleCell& cell1, ParticleCell& cell2,
 				double dd = molecule2.dist2(molecule1, dummy);
 				if (dd < _cutoffRadiusSquare) {
                     _rdf->observeRDF(molecule1, molecule2, dd);
+					if (_rdf->doARDF() == true) {
```

Comment:
```suggestion
					if (_rdf->doARDF()) {
```
simplify this statement